app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = list()
+
+    for person_info in people:
+        person_entry = Person(name=person_info["name"], age=person_info["age"])
+        people_list.append(person_entry)
+
+    for person in people:
+        if person.get("husband") is not None:
```

Comment:
`is not None` redundant here, since in your case `if` statement will be `True` if key exists